src/Robo/Plugin/Commands/CICommands.php

```diff
@@ -91,12 +99,15 @@ public function jobCheckCodingStandards(): Result
         $standards = implode(',', $this->getCodingStandards());
         $extensions = implode(',', $this->phpcsCheckExtensions);
         $ignorePaths = implode(',', $this->phpcsIgnorePaths);
-        return $this->taskExecStack()
-            ->stopOnFail()
-            ->exec('vendor/bin/phpcs --config-set installed_paths vendor/drupal/coder/coder_sniffer')
-            ->exec("vendor/bin/phpcs --standard=$standards --extensions=$extensions \
-                --ignore=$ignorePaths $this->customCodePaths")
-            ->run();
+        /** @var \Robo\Task\CommandStack $stack */
+        $stack = $this->taskExecStack()->stopOnFail();
+        $stack->exec('vendor/bin/phpcs --config-set installed_paths vendor/drupal/coder/coder_sniffer');
+        $stack->exec("vendor/bin/phpcs --standard=$standards --extensions=$extensions \
+                --ignore=$ignorePaths $this->customCodePaths");
+        if ($this->lintTwigFiles) {
+            $stack->exec("vendor/bin/twig-cs-fixer lint $this->customCodePaths");
```

Comment:
I am also thinking about it if it would be better to break into two separate jobs but that is not a blocker for this. Need to think on it further.

Reply:
I like that we are thinking about this.

I guess I never even thought about this, as I only interact with the end command of `composer cs-check`, which makes no mention of PHP or Twig.

Does that change your opinion?

I went ahead and added the alias for now though.

Reply:
`composer cs-check` is a repo shortcut to call this job, so it is 1:1. I am trying to consider scenarios where someone working locally might want to lint twig without linting PHP either for speed reasons or because they know PHP is in a failed state. The way this is currently implemented if an earlier step (like PHP linting) fails it will skip the rest.

Reply:
I also just realized in my testing that implemented in this way, twig linting will run on this repo, which doesn't use twig. Again, not a blocker, but that might tip me over to splitting it up. 🤔

Reply:
Ya, I can see how it would be good to separate it. Let me sort out the other issues then we can decide if we do that in this PR or in a follow-up one.

Reply:
The blocker in the CHQ PR has been resolved and I have added the ability to toggle Twig linting. Are we good to resolve this conversation?